scipy/interpolate/_cubic.py

```diff
@@ -101,6 +101,24 @@ class CubicHermiteSpline(PPoly):
         based on first and last intervals, or to return NaNs. If 'periodic',
         periodic extrapolation is used. If None (default), it is set to True.
 
+    See Also
```

Comment:
based on https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring, I think the order should be:
- Parameters
- Attributes
- Methods
- See Also
- Notes
- References

so it looks like it was already correct before your change? Do you agree?

Reply:
I was confused by the same earlier, but apparently the numpydoc tooling reorders the sections regardless, so the rendered documentation would be in the same order as before, i.e. the HTML wouldn't change. The changes in the source were thus made for consistency with the rendered docs.

Reply:
Ah, yeah, that looks like a bug. How about you revert the change here, and I'll open an issue about how the docs are rendering currently? Unless Ralf thinks otherwise

Reply:
Thanks, just reverted the commit!